_posts/2015-10-19-quickstart-install-kissmetrics.md

```diff
@@ -0,0 +1,181 @@
+
+###Step 3 - Insert the Tracking Code
+We've copied our JavaScript tracking code and now we need to edit our website's source code. To start, we'll first locate the <head> tag at the top:
+
+![Install Kissmetrics][3]
+
+Make some space for it and paste the tracking code:
+
+![Install Kissmetrics][4]
+
+That's all there is to it! After it is live on your server, Kissmetrics will automatically start sending you some events.
+
+##What the Kissmetrics JavaScript Snippet Tracks Automatically
+Once you install the tracking code on your site, you will start to see your data within the hour. 
+
+Some [events and properties](https://blog.kissmetrics.com/people-events-and-properties/) are automatically tracked and will start appearing when the data comes in:
+
+**Event: Visited Site**
+This event triggers when a person visits any page of your site that has the JavaScript tracking code. Along with this information, the properties Referrer, URL, and Returning are tracked. 
+
+**Referrer** records the URL the person came from. This property is particularly useful for learning where your customers came from. Using a Funnel Report, you can segment visitors by referrer to learn which referrers not only bring the most visitors, but more importantly which referrers bring the most customers. 
+
+**URL** tells you which URL a new visitor first landed on. With the Funnel Report, you'll be able to segment people by this URL and test pages against each other to see which ones led to conversion. 
+
+**Returning** triggers when the person comes back to your site. If the person returns, they'll be assigned a 1 next to the property. The time between visits must be 30 minutes or longer for the Returning property to trigger. If they visit your site, leave and return 25 minutes later, Returning won't trigger. At least 30 minutes must elapse for "Returning" to set. When looking at the person details report, you'll see how many times the person has returned:
+
+![Install Kissmetrics][5]
+
+*The "x13" box next to the Returning property indicates the person has returned to our site 13 times since their first visit. When we hover over the box we see when Kissmetrics first saw them return as well as when they most recently returned.* 
+
+You can segment people by their Returning property in a Funnel Report. Here's how that looks:
+
+![Install Kissmetrics][6]
+
+Here we see a typical signup funnel; we're tracking people who visit our site, view the features page and signup. People are segmented by the Returning property. We're seeing how new vs returning visitors convert through the funnel. While most of the people that come to our site are new visitors, we actually get more signups from returning visitors. This tells us that most people make more than one visit before they decide to sign up. 
+
+**Event: Ad Campaign Hit**
```

Comment:
Seems like you want a `<br>` here to match the two events below.

So: `**Event: Ad Campaign Hit**<br>`